### DISCUSSION TITLE
Redesign Landing Page

### DISCUSSION THREAD

**Discussion by dev-shetty** [karmacircleCommunity/KarmaCircle#648] [Ideas]
### The landing page looks too simple now, the landing page should always be creative and attention grabber of the user

If anyone has a cool idea to improve it will be great for the first impression of the website.

If anyone has a good idea, drop it down here.

**Comment by dev-shetty** [comment]
My idea may be changing the color of important text and use some text decoration, so that the user mainly focus on that.

**Comment by tamalCodes** [comment]
Great one, let's see if we can get a good designer who'll make something, if you want to make something you can maybe either code it out somewhere, on a separate branch, or we'll get to have a designer who can do this better 🔥

**Reply by dev-shetty** [reply]
Yeah...sounds great 🚀


Will try my best 🙌
